internal/cmd/oohelperd/oohelperd.go

```diff
@@ -10,19 +10,22 @@ import (
 
 	"github.com/apex/log"
 	"github.com/ooni/probe-cli/v3/internal/cmd/oohelperd/internal"
+	"github.com/ooni/probe-cli/v3/internal/cmd/oohelperd/internal/nwcth"
 	"github.com/ooni/probe-cli/v3/internal/engine/netx"
 )
 
 const maxAcceptableBody = 1 << 24
 
 var (
-	dialer    netx.Dialer
-	endpoint  = flag.String("endpoint", ":8080", "Endpoint where to listen")
-	httpx     *http.Client
-	resolver  netx.Resolver
-	srvcancel context.CancelFunc
-	srvctx    context.Context
-	srvwg     = new(sync.WaitGroup)
+	dialer     netx.Dialer
+	endpoint   = flag.String("endpoint", ":8080", "Endpoint where to listen")
+	httpx      *http.Client
+	httpnx     *http.Client
+	quicdialer netx.QUICDialer
+	resolver   netx.Resolver
+	srvcancel  context.CancelFunc
+	srvctx     context.Context
+	srvwg      = new(sync.WaitGroup)
```

Comment:
It's unclear to me whether the code you added here still matters. It seems it does not since the only other change below is a new `mux.Handle` line. Am I missing something?